src/VisualStudio/Core/Impl/ProjectSystem/CPS/CPSProjectFactory.cs

```diff
@@ -95,7 +95,6 @@ IWorkspaceProjectContext IWorkspaceProjectContextFactory.CreateProjectContext(st
                 await TaskScheduler.Default;
             }
 
-            // CPSProject constructor has a UI thread dependencies currently, so switch back to the UI thread before proceeding.
```

Comment:
can we update:

```
            var visualStudioProject = await _projectFactory.CreateAndAddToWorkspaceAsync(
                projectUniqueName, languageName, creationInfo, cancellationToken).ConfigureAwait(true);
```

to be ConfigAwait(false) now?

Reply:
Ah yes, good catch.